build-it/build.py

```diff
@@ -69,22 +76,37 @@ def copy_binaries_for_target(self):
             os.makedirs(generator_dll_destination)
         distutils.file_util.copy_file(generator_dll_source, generator_dll_destination)
 
-    def copy_built_binaries(self):
-        if self.build_target == "All":
-            self.copy_binaries_all_targets()
+    def copy_built_binaries(self, args):
+        if args.target == "All":
+            self.copy_binaries_all_targets(args)
         else:
             self.copy_binaries_for_target()
```

Comment:
I guess you have to pass args here as well. Also update the code the method copy_binaries_for_target() to use args